install.sh

```diff
@@ -170,33 +172,33 @@ NAMESPACE_DEFAULT="liqo"
 LIQO_VERSION_DEFAULT="latest"
 LIQO_SUFFIX_DEFAULT=""
 
-# Necessary Commands
+# Check necessary commands to be installed before installing Liqo
 commands="curl kubectl"
 
 echo "[PRE-CHECK]: Checking all pre-requisites are met"
 for val in $commands; do
-    if command -v $val > /dev/null; then
-      echo "[PRE-CHECK]: $val correctly found"
+    if command -v "$val" > /dev/null; then
+      echo "[PRE-INSTALL]: $val correctly found"
```

Comment:
Is this intentional or left by the rebase?

The same for all the following `[PRE-INSTALL]` modifications

Reply:
Yes, I thought we would like to move to PRE-INSTALL.